compiler/src/bin/native/serve.ml

```diff
@@ -51,7 +54,7 @@ let html_source =
 </body>
 </html>
   |html}
-    [%blob "compiler/src/bin/native/client/client.bc.js"]
+    [%blob "client/client.bc.js"]
```

Comment:
Hmm it seems to work in both situation, is it some `blob` ppx magic? Anyway, the short form is better.

Reply:
The long form _doesn't_ work when you vendor (in the dune sense) slipshow ^^"

Reply:
Ah yes! I understand. Thanks for the explanation :)